ycms/cms/views/zentral.py

```diff
@@ -0,0 +1,51 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+from ..models import Ward , Patient, User
+from django.template import loader
+from django.shortcuts import render
+from django.http import Http404
+from django.shortcuts import get_object_or_404, render
+from django.views.generic import TemplateView
+
+
+class ZentralView(TemplateView):
+    """
+    View to see all wards data and add a new one
+    """
+    template_name = "zentral.html"
+
+    def newward(self, request):
```

Comment:
```suggestion
    def get(self, request, *args, **kwargs):
```

I haven't had a chance to take a look from my PC yet, but this probably won't work, since Django will expect you to implement a get method